astropy/coordinates/representation.py

```diff
@@ -2339,14 +2339,16 @@ def from_cartesian(cls, other, base):
         ----------
         other :
             The object to convert into this differential.
-        base : instance of ``self.base_representation``
+        base : `BaseRepresentation`
              The points for which the differentials are to be converted: each of
              the components is multiplied by its unit vectors and scale factors.
+             Will be converted to ``self.base_representation`` if needed.
```

Comment:
Nitpick, but should this be 
```suggestion
             Will be converted to ``cls.base_representation`` if needed.
```
?

Reply:
OK, thanks!  I made the change as part of rebasing - might as well check that more tests pass now...